_data/crates.yaml

```
@@ -229,6 +232,9 @@
 
 - name: rurel
   topics: ["reinforcement"]
+
+- name: rust-bert
+  topics: ["NLP"]
```

Tags are in lowercase, so this should probably be lowercase too. Also, you should create a page at `_topics/nlp.md` to display this entry on.
```suggestion
  topics: ["nlp"]
```

---

@matiu2 Per aidan-fitz's review, the landing page for your pull request doesn't have a new tile for the Natural Language Processing page. You can add a file `nlp.md` to the `/_topics` directory, using the other topics there as as a template. Not sure about how to do the formatting such that it doesn't end up on a new line, but we can cross that bridge when the topic's put together to your satisfaction